src/Microsoft.DotNet.Configurer/LocalizableStrings.resx

```diff
@@ -118,9 +118,9 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="FirstTimeMessageWelcome" xml:space="preserve">
-    <value>Welcome to .NET Core 3.0!
+    <value>Welcome to .NET Core {0}!
```

Comment:
I'm not sure if you can add parameters to an existing string. It's probably OK but check with @nguerrera or someone familiar with localization.  Also if you do add parameters I'm not sure if you should change what `{0}` means, maybe the new data should be in `{1}`.

Reply:
should be ok since the xlf has `<target state="new"` . But ya, wait for Nick to double check

Reply:
Yes, it reverts to English now when the placeholders change. :)